### PR TITLE
feat: Load multiple candidate streams concurrently

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

**Review by oed — COMMENTED** [review]
Not super familiar with the code here. Are we fine with these changes without any tests? I guess we will test this in dev / clay.

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -171,9 +166,22 @@ export default class AnchorService {
...
+      conflictingRequestCount: groupedRequests.conflictingRequests.length,
+      failedRequestsCount: groupedRequests.failedRequests.length,
+      failedToPublishAnchorCommitCount:
+        groupedRequests.acceptedRequests.length - numAnchoredRequests,
+      unprocessedRequestCount: groupedRequests.unprocessedRequests.length,
+      candidateCount: candidates.length,
+      anchorCount: anchors.length,
+    })
+
+    await Utils.delay(5000)
```
[the remark this answers is not in the document]
Comment (stbrody) [reply src/services/anchor-service.ts:184]: yeah, it's still to flush logs.  I actually already put back a comment clarifying that locally

File: src/services/__tests__/anchor-service.test.ts [hunk 2]
```diff
@@ -482,7 +482,7 @@ describe('anchor service', () => {
         ceramicService.putStream(streamId, stream)
       }
 
-      const candidates = await anchorService._findCandidates(requests, 0)
+      const [candidates, _] = await anchorService._findCandidates(requests, 0)
```
Comment (ukstv) [inline src/services/__tests__/anchor-service.test.ts:485]:
```suggestion
      const [candidates] = await anchorService._findCandidates(requests, 0)
```

Unless I miss something, there is no need to have temporary "_" variable.

**Review by ukstv — APPROVED** [review]
Seems all right'ish in large, too complicated in small, but if it works, it better be out. Even with temporary variables.

**Comment by stbrody** [comment]
Merged develop branch into this PR, in prep to merge this to develop on Monday

**Comment by stbrody** [comment]
We are keeping up without this.  Closing this for now to avoid potentially destabilizing anything or causing more anchors to fail due to extra load.  If we need to speed up anchoring in the future we can re-consider this

**Pull request closed without merging**